aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -1536,10 +1548,6 @@ func updateExtendedS3Config(d *schema.ResourceData) *firehose.ExtendedS3Destinat
 		configuration.CloudWatchLoggingOptions = extractCloudWatchLoggingConfiguration(s3)
 	}
 
-	if v, ok := s3["error_output_prefix"]; ok && v.(string) != "" {
```

Comment:
This is actually a bug.  Because it checks that `v.(string) != ""`, you cannot clear the `ErrorOutputPrefix` after setting it.  The plan shows it, but the apply step never updates it.